[bitnami/redis] Add new environment variable REDIS_TLS_CA_DIR to support tls-ca-cert-dir config in redis.conf (#56375)
bitnami/containers: bitnami/redis/README.md

* `REDIS_TLS_CERT_FILE`: File containing the certificate file for the TLS traffic. No defaults.
* `REDIS_TLS_KEY_FILE`: File containing the key for certificate. No defaults.

> * `REDIS_TLS_CA_FILE`: File containing the CA of the certificate. No defaults.
> * `REDIS_TLS_CA_DIR`: Directory containing the CA certificates. No defaults.
> * `REDIS_TLS_DH_PARAMS_FILE`: File containing DH params (in order to support DH based ciphers). No defaults.
> * `REDIS_TLS_AUTH_CLIENTS`: Whether to require clients to authenticate or not. Defaults to `yes`.
> 
> When both `REDIS_TLS_CA_FILE` and `REDIS_TLS_CA_DIR` are set, `REDIS_TLS_CA_FILE` is used preferentially.
> 

Reviewer: ```suggestion
* `REDIS_TLS_CA_FILE`: File containing the CA of the certificate (takes precedence over `REDIS_TLS_CA_DIR`). No defaults.
* `REDIS_TLS_CA_DIR`: Directory containing the CA certificates. No defaults.
* `REDIS_TLS_DH_PARAMS_FILE`: File containing DH params (in order to support DH based ciphers). No defaults.
* `REDIS_TLS_AUTH_CLIENTS`: Whether to require clients to authenticate or not. Defaults to `yes`.
```
